picard/ui/util.py

```diff
@@ -104,7 +104,7 @@ def __init__(self, *args):
         super(MultiDirsSelectDialog, self).__init__(*args)
         self.setFileMode(self.Directory)
         self.setOption(self.ShowDirsOnly)
-        if sys.platform == "darwin":
+        if sys.platform in ["darwin", "win32"]:
```

Comment:
I have no idea what happens on Linux. All I know is that Windows used to get non-native dialog and in QT5 it didn't - so I fixed it for Windows and left it for Linux because I had no knowledge to do anything different.

As noted, Windows native dialog doesn't provide multi-select under Qt4 or Qt5. Qt4 apparently gave non-native dialog as standard whilst Qt5 doesn't.